nni/compression/pytorch/speedup/compressor.py

```diff
@@ -71,10 +76,12 @@ def __init__(self, model, dummy_input, masks_file, map_location=None,
         elif isinstance(masks_file, dict):
             self.masks = masks_file
         else:
-            raise Exception('Please provide the mask or the path of the mask file')
+            raise Exception(
+                'Please provide the mask or the path of the mask file')
         self.constant = {}
         # self.internal_result save the internal output of the submodules
         self.internal_result = {}
+        self.customized_replace_func = customized_replace_func
```

Comment:
maybe: `self.customized_replace_func = customized_replace_func if customized_replace_func is not None else {}`. then at the bottom, no need to check self.customized_replace_func any more

Reply:
good suggestion!